Modules/Microsoft365DSC/DSCResources/MSFT_AADAuthenticationMethodPolicySms/MSFT_AADAuthenticationMethodPolicySms.schema.mof

```diff
@@ -0,0 +1,28 @@
+[ClassVersion("1.0.0")]
+class MSFT_MicrosoftGraphExcludeTarget2
```

Comment:
To avoid conflicts across resources in the future, we took the decision to always rename these CIMInstances as:

MSFT_resourceNameCIMInstance

In this case, it should be MSFT_AADAuthenticationMethodPolicySMSExcludeTarget

Please review every schema.mof and module files in this Pull Request to adhere to this naming standard. Thanks